.github/workflows/sizecheck-report.yaml

```diff
@@ -0,0 +1,40 @@
+name: Report Sizecheck Results
+
+on:
+  workflow_run:
+    workflows: [Sizecheck]
+    branches: ['**']
+    types: [completed]
+
+jobs:
+  on-success:
+    name: Report sizecheck success
+    runs-on: ubuntu-latest
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
+    steps:
+      - name: Comment on size report empty
+        uses: thollander/actions-comment-pull-request@v2
+        with:
+          message: 'The size of lit-html.js and lit-core.min.js are as expected.'
+          comment_tag: check-size # ensures we overwrite old failure comment if present
+
+  on-failure:
+    name: Report sizecheck failure
+    runs-on: ubuntu-latest
+    if: ${{ github.event.workflow_run.conclusion == 'failure' }}
+    steps:
+      # Download the artifact from the triggering workflow that contains the
+      # size check results to report
+      - name: Download artifact
+        uses: dawidd6/action-download-artifact@v2
+        with:
+          workflow: ${{ github.event.workflow.id }}
+          run_id: ${{ github.event.workflow_run.id }}
+          name: sizecheck
+          path: scripts
```

Comment:
Should this path be the markdown file? `scripts/check-size-out.md`?

Reply:
This one specifies the directory location where the artifact files are unzipped into. I think omitting it would be okay here too which will unzip the files at the root and next step could just find the file from the root, but I just decided to keep the directory name that it had from the workflow for no particular reason.

Reply:
If the path here is scripts and the path for the uploaded file is scripts/check-size-out.md will that write the file to scripts/scripts/check-size-out.md?

Reply:
No. The upload artifact action here https://github.com/lit/lit/blob/04d16b181c292f0951097fce691ea182bb14b29c/.github/workflows/sizecheck.yaml#L27-L32
specifies the path to the file, and the individual file will be uploaded without the directory structure. https://github.com/actions/upload-artifact#upload-an-individual-file

Reply:
We do the same thing for benchmarks here with paths to individual files
https://github.com/lit/lit/blob/cabe72a863a5de14a4bbca384374db748dd9b4c5/.github/workflows/benchmarks.yml#L86-L95
that are downloaded to `results` directory and read from there
https://github.com/lit/lit/blob/cabe72a863a5de14a4bbca384374db748dd9b4c5/.github/workflows/benchmarks-report.yaml#L33-L48